home/templates/home/home_page.html

```diff
@@ -96,6 +96,41 @@ <h1 class="leaderboard-header align-center">Leaderboard</h1>
     </div>
   </section>
 
+  {% if regions|length > 0 %}
+    <section class="leaderboard-section">
+      <h1 class="leaderboard-header align-center">View Leaderboards by Category</h1>
+      <div class="d-flex justify-center ">
+        {% for region in regions|slice:":5" %}
+          <div class="align-center">
+            <a href="{% url 'sites:leaderboard' slug=region.slug %}">
+              <div class="sm-button button-gray" aria-role="button">
+                {{ region.name }} <i class="fa fa-chevron-right" aria-hidden="true"></i>
+              </div>
+            </a>
+          </div>
+        {% endfor %}
+      </div>
```

Comment:
It's possible there needs to be some more whitespace in the styles for this `<div>`. When you have five or fewer regions, this is what the bottom of the home page looks like on my end:

![image](https://user-images.githubusercontent.com/561931/64994012-825f5500-d8a5-11e9-896d-0b050c8970f1.png)

Where the region buttons feel very close to the top of the footer to me.

When there are more than 5 regions the spacing feels correct to me.